src/components/Header/index.tsx

```diff
@@ -4,13 +4,25 @@ import * as S from './style';
 
 import { LogoIcon } from '@/assets';
 
-const Header = () => (
-  <S.Header>
-    <S.Inner>
-      <LogoIcon />
-      <S.PageName>취업자 리스트 조회</S.PageName>
-    </S.Inner>
-  </S.Header>
-);
+interface Props {
+  clearList: () => void;
+}
+
+const Header: React.FC<Props> = ({ clearList }) => {
+  const onClick = () => {
+    clearList();
+  };
+
+  return (
+    <S.Header>
+      <S.Inner>
+        <S.LogoButton onClick={onClick}>
```

Comment:
```suggestion
  return (
    <S.Header>
      <S.Inner>
        <S.LogoButton onClick={clearList}>
```

onClick 함수 내부가 간단하여 바로 적어줘도 괜찮다고 생각합니다

Reply:
반영하겠습니다!